packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -81,12 +81,6 @@ export type CreateChildSummarizerNodeParam = {
 export interface ISummarizerNode {
     /** Latest successfully acked summary reference sequence number */
     readonly referenceSequenceNumber: number;
-    /**
-     * True if a change has been recorded with sequence number exceeding
-     * the latest successfully acked summary reference sequence number.
-     * False implies that the previous summary can be reused.
-     */
-    hasChanged(): boolean;
```

Comment:
Was this just removed because it was unused?

Reply:
That's one of the reasons. But the main reason for not exposing it is because it would return incorrectly if called before the reference state is loaded. For SummarizerNode, the referenceSequenceNumber is part of the latest summary which is provided during creation.
However, for SummarizerNodeWithGC, the initial referenced used state / route is obtained asynchronously when summarizing for the first time. If hasChanged is called before that, it would always return true.

Reply:
That is a good explanation and reasoning, thanks!